src/Ledger/Gov.lagda

```diff
@@ -236,7 +235,7 @@ maxAllEnactable e = maxsublists⊧P (allEnactable? e)
 \label{defs:enactable}
 \end{figure*}
 
-\GovState behaves similar to a queue. New proposals are appended at
+\GovState behaves like a queue. New proposals are appended at
```

Comment:
...so, it's a [priority queue](https://en.wikipedia.org/wiki/Priority_queue)?

Reply:
No, it's not that either, since it's completely unknown what element is going to be removed next because it depends on outside information. Though note that the article also says 'similar to a regular queue'.

Reply:
Okay, I mainly changed it initially because I didn't like the grammar.  I'll change it to "The behavior of \GovState is similar to that of a queue."